docs/deployment/helm.md

```diff
@@ -152,28 +152,29 @@ We have introduced the following Chart parameters.
+| `topologyUpdater.podSecurityContext`          | dict   | {}                      | [PodSecurityContext](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod) holds pod-level security attributes and common container settings |
+| `topologyUpdater.securityContext`             | dict   | {}                      | Container [security settings](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container)                                                    |
+| `topologyUpdater.resources`                   | dict   | {}                      | Topology updater pod [resources management](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/)                                                                           |
+| `topologyUpdater.nodeSelector`                | dict   | {}                      | Topology updater pod [node selector](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector)                                                                           |
+| `topologyUpdater.tolerations`                 | dict   | {}                      | Topology updater pod [node tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)                                                                                |
+| `topologyUpdater.annotations`                 | dict   | {}                      | Topology updater pod [annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)                                                                                    |
+| `topologyUpdater.affinity`                    | dict   | {}                      | Topology updater pod [affinity](https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity/)                                                                      |
+| `topologyUpdater.config`                      | dict   |                         | [configuration](../reference/topology-updater-configuration-reference)                                                                                                                                |
+| `topologyUpdater.podSetFingerprint`           | bool   | false                   | Enables compute and report of pod fingerprint in NRT objects.                                                                                                                                         |
+| `topologyUpdater.kubeletStateDir`             | string | "/host-var/lib/kubelet" | Specifies kubelet state directory path for watching state and checkpoint files                                                                                                                        |
```

Comment:
Mention that empty value disables kubelet state tracking(?)

Reply:
Added. Thank you